docker-compose.yml

```diff
@@ -7,8 +7,8 @@ services:
     build:
       context: .
       dockerfile: UT4MasterServer/Dockerfile
+    hostname: api.ut4uu.com
```

Comment:
This should be configurable and default should be umm you know, default value ;) (localhost?)

Reply:
Localhost is always served, specifying hostname does not prevent that. Dockerfiles are configuration, if you mean it should be split by region that is an option.

Reply:
Not sure how long you've been developing for, but you should work on adding less snark to your PR comments. It makes you look like less of a tool when your suggestion is dead wrong.

Reply:
> Localhost is always served

OK